ipasetup.py.in

```diff
@@ -19,20 +19,21 @@ import sys
 
 
 PACKAGE_VERSION = {
-    'cryptography': 'cryptography >= 0.9',
+    'cryptography': 'cryptography >= 1.5',
     'dnspython': 'dnspython >= 1.11.1',
-    'gssapi': 'gssapi > 1.1.2',
+    'gssapi': 'gssapi >= 1.2.0',
```

Comment:
You are treating PyPI packages like you are treating RPM packages. That is not how it works. Distribution packages have curated, well test combinations of package versions. Do you have enough resources to test the packages with every release of python-cryptography since 0.9?